bigbluebutton-html5/private/config/settings.yml

```diff
@@ -33,6 +33,8 @@ public:
     copyright: '©2022 BigBlueButton Inc.'
     html5ClientBuild: HTML5_CLIENT_VERSION
     helpLink: https://bigbluebutton.org/html5/
+    iosAppStoreUrl: ''
```

Comment:
Change to this value: https://apps.apple.com/us/app/bigbluebutton-tablet/id1641156756